pkg/controller.v1beta1/experiment/manifest/generator_test.go

```diff
@@ -88,23 +88,18 @@ func TestGetRunSpecWithHP(t *testing.T) {
 		t.Errorf("ConvertObjectToUnstructured failed: %v", err)
 	}
 
-	tcs := []struct {
-		Instance             *experimentsv1beta1.Experiment
-		ParameterAssignments []commonapiv1beta1.ParameterAssignment
-		expectedRunSpec      *unstructured.Unstructured
-		Err                  bool
-		testDescription      string
+	cases := map[string]struct {
+		Instance                       *experimentsv1beta1.Experiment
+		ParameterAssignments           []commonapiv1beta1.ParameterAssignment
```

Comment:
```suggestion
		instance                       *experimentsv1beta1.Experiment
		parameterAssignments           []commonapiv1beta1.ParameterAssignment
```

I found these fields are exported. We should not export test fields at all points.
But this is irrelevant to this PR. So, could you open dedicated PR so that we can avoid to export test fields.

Reply:
I can open a separate PR once this one is merged.